packages/e2e-test-utils/src/keyboard-mode.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Move the mouse to trigger Edit Mode
```

Comment:
This description draws my attention assumptive of the use of a cursor. Is there a particular reason we can't use a keyboard event to make the switch?

**Edit:** I guess it's because we don't know for sure that we're in edit or navigation mode, and it's used primarily as a "blanket" reset? That sort of description seems more useful here.

```suggestion
 * Triggers edit mode if not already active.
```

I guess the alternative is detecting that we're in navigation mode (either via presence of some DOM or by a selector call) and pressing <kbd>Enter</kbd>?

Reply:
>  Is there a particular reason we can't use a keyboard event to make the switch?

The keyboard event only works if we're focusing the block toolbar. This is a way to trigger the edit mode at the beginning of a test for instance without focusing any particular block.